web/src/stream_settings_ui.js

```diff
@@ -796,10 +796,15 @@ export function launch(section, left_side_tab, right_side_tab) {
 
 export function switch_rows(event) {
     const active_data = stream_settings_components.get_active_data();
+    const $add_subscriber_pill = $(".add_subscribers_container > div > .input");
```

Comment:
I think this variable can be renamed to `$add_subscriber_pill_input` or `$add_subscriber_pill_container`. And I think the selector can be just simplified to be `$(".add_subscribers_container .input")`.

Reply:
I see, the simplified selector works! thanks